pkgs/tools/networking/p2p/libtorrent-jesec/default.nix

```diff
@@ -2,20 +2,19 @@
 
 stdenv.mkDerivation rec {
   pname = "libtorrent-jesec";
-  version = "0.13.8-r1";
+  version = "0.13.8-r2";
 
   src = fetchFromGitHub {
     owner = "jesec";
     repo = "libtorrent";
     rev = "v${version}";
-    sha256 = "sha256-Eh5pMkSe9uO0dPRWDg2BbbRxxuvX9FM2/OReq/61ojc=";
+    sha256 = "sha256-eIXVTbVOCRHcxSsLPvIm9F60t2upk5ORpDSOOYqTCJ4=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ openssl zlib ];
 
-  # https://github.com/jesec/libtorrent/issues/1
-  doCheck = false;
+  doCheck = true;
```

Comment:
As of this release, tests can now be re-enabled. However, there may be an issue with them not being picked up at all (see https://github.com/jesec/libtorrent/issues/1#issuecomment-851100316)

No matter if this behavior is intended or not, should tests be disabled until running the tests isn't a no-op, even if it doesn't error?

Reply:
Can you fetchpatch https://github.com/jesec/libtorrent/commit/a38205ce06aadc9908478ec3a9c8aefd9be06344? Otherwise we think we have test coverage when we really don't have it.

Reply:
Done.